SmartDeviceLinkTests/Notifications/SDLResponseDispatcherSpec.m

```diff
@@ -109,7 +109,7 @@
             });
             
             it(@"should run the handler", ^{
```

Comment:
I'm still getting random test cases failing locally. Travis is reporting failed test cases in this class as well.